internal/command/clusters/uninstall.go

```diff
@@ -0,0 +1,218 @@
+package clusters
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"strings"
+	"time"
+
+	certmanagerv1 "github.com/cert-manager/cert-manager/pkg/apis/certmanager/v1"
+	"github.com/spf13/cobra"
+	corev1 "k8s.io/api/core/v1"
+
+	"github.com/jetstack/jsctl/internal/command/types"
+	"github.com/jetstack/jsctl/internal/kubernetes"
+	"github.com/jetstack/jsctl/internal/kubernetes/clients"
+)
+
+func Uninstall(run types.RunFunc, kubeConfigPath *string) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "uninstall",
+		Short: "Contains commands to check a cluster before the uninstallation of Jetstack Secure software",
+	}
+
+	cmd.AddCommand(verify(run, *kubeConfigPath))
+
+	return cmd
+}
+
+func verify(run types.RunFunc, kubeConfigPath string) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "verify",
+		Short: "Check that a cluster is ready to have Jetstack Software uninstalled",
+		Args:  cobra.MatchAll(cobra.ExactArgs(0)),
+		Run: run(func(ctx context.Context, args []string) error {
+			kubeCfg, err := kubernetes.NewConfig(kubeConfigPath)
+			if err != nil {
+				return err
+			}
+
+			// first, we need to check that secrets containing certs are safe from
+			// garbage collection
```

Comment:
Suggestion: print out which check is being run

Reply:
Fixed in 936699fba6bcae721a7f764ed376476a949d2d72